test/core/derivatives/implementation/base.py

```diff
@@ -43,28 +43,17 @@ def jac_t_mat_prod(self, mat: Tensor) -> Tensor:
         raise NotImplementedError
 
     @abstractmethod
-    def weight_jac_t_mat_prod(
-        self, mat: Tensor, sum_batch: bool, subsampling: List[int] = None
+    def param_jac_t_mat_prod(
```

Comment:
Rename to `param_mjp` to improve consistency

Reply:
Yes.